tests/integration/api/test_decision_environment.py

```diff
@@ -327,6 +327,189 @@ def test_create_decision_environment_with_empty_credential(
         assert status_message in str(errors)
 
 
+@pytest.mark.parametrize(
```

Comment:
Suggestion: 
It seems that the test parameters are basically the same across both tests. The only difference is the status code, in POST is 201, and in PATCH 200. What do you think if the parameter has only a generic variable instead of the status code, for example a boolean called "expected_sucess" and determine inside the test what is the expected return code for a successful or failed request. In that way you can reuse the same set of parameters across both tests.

Reply:
Good point, I just added this